cmd/goal/clerk.go

```diff
@@ -211,6 +213,17 @@ var sendCmd = &cobra.Command{
 		fromAddressResolved := accountList.getAddressByName(account)
 		toAddressResolved := accountList.getAddressByName(toAddress)
 
+		// Parse lease field
+		leaseBytesRaw, err := base64.StdEncoding.DecodeString(lease)
+		if err != nil {
+			reportErrorf(malformedLease, lease, err)
+		}
+		if len(leaseBytesRaw) != 32 {
+			reportErrorf(malformedLease, lease, fmt.Errorf("lease length %d != 32", len(leaseBytesRaw)))
+		}
+		var leaseBytes [32]byte
+		copy(leaseBytes[:], leaseBytesRaw)
```

Comment:
Given that the content of the `lease` variable is optional ( i.e. could be empty ), I think that you don't really want to fail here if the encoded size isn't 32 bytes long.
I think that only the expect tests overs this functionality.

Reply:
Ah, you're right--the field will be empty if not set. Forgot about this--I'll go fix it.